includes/Traits/Types.php

```diff
@@ -108,4 +108,34 @@ public function get_allowed_mime_types() {
 
 		return $allowed_mime_types;
 	}
+
+	/**
+	 * Returns a list of image mime types.
+	 *
+	 * @since 1.4.0
+	 *
+	 * @return array List of allowed mime types.
+	 */
+	public function get_allowed_image_mime_types() {
+		$mime_type         = $this->get_allowed_mime_types();
+		$allowed_mime_type = $mime_type['image'];
```

Comment:
If the SVG flag is enabled, this will add SVGs to this list, which is what we don't want.

Reply:
What is what the `array_intersect` is for 😄

Reply:
https://github.com/google/web-stories-wp/pull/6074/files/570cef5bf94c51b37b298e8ffdddcdd2c1decb72#diff-f44eb7e6dbedc04516997b6e79d04e2b3e72388ca59d3a76c39ee19d90f6a70eR139

Reply:
Ah, seeing it now!

So if someone removes support for PNG via the `web_stories_allowed_mime_types` filter, it would also be removed here?

And if someone wants to add support for more images they'd need to use both filters?

Reply:
> And if someone wants to add support for more images they'd need to use both filters?

Yes, this is by design. If you add support for svg yourself for example, we don't want that filter down to allow images. You need to manually add it. It is basically a whitelist of allowed image types. We easily be flipped to be a black list. What do you think is the best developer expereience here?

Reply:
Not sure what the best experience would be, but the current solution seems reasonable for sure.